### PR TITLE
Fix rounding discrepancies between VB and C++

### PR DESCRIPTION
**Pull request by denravonska**
This is how the C++ hashes are calculated, as opposed to the default round-to-even in VB.NET.

Without the fix the hashes will be calculated differently in VB and C++:
```
"avg" : 214901.03302348,
"beacon_count" : 2792.00000000,
"avg_mag" : 64.30597788,
"beacon_participant_count" : 2761.00000000,
"superblock_valid" : true,
".NET Neural Hash" : "2ca1ca84321f09a22bd74825d3aa727b",
"Length" : 71587.00000000,
"Wallet Neural Hash" : "e0edcfd93842c120b7e0656c39d310dc"
```

With the fix:
```
"avg" : 214901.03302348,
"beacon_count" : 2790.00000000,
"avg_mag" : 64.30597788,
"beacon_participant_count" : 2595.00000000,
"superblock_valid" : true,
".NET Neural Hash" : "e0edcfd93842c120b7e0656c39d310dc",
"Length" : 70757.00000000,
"Wallet Neural Hash" : "e0edcfd93842c120b7e0656c39d310dc"
```

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -80,3 +80,9 @@ Gridcoin is released under the terms of the MIT license. See [COPYING](COPYING)
...
+|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| [![Build Status](https://travis-ci.org/gridcoin/Gridcoin-Research.svg?branch=development)](https://travis-ci.org/gridcoin/Gridcoin-Research) | [![Build Status](https://travis-ci.org/gridcoin/Gridcoin-Research.svg?branch=staging)](https://travis-ci.org/gridcoin/Gridcoin-Research) | [![Build Status](https://travis-ci.org/gridcoin/Gridcoin-Research.svg?branch=master)](https://travis-ci.org/gridcoin/Gridcoin-Research) |
```
Comment (tomasbrod) [inline README.md:88]: Why the readme is in all your commits?

**Review by tomasbrod — APPROVED** [review]
This is semi mandatory: mandatory for windows NN participants with moderately high balance and leisure for everyone else.

File: README.md (hunk 1 above)
Comment (denravonska) [reply README.md:88]: I have no idea. It's also in master (and all the other branches) so it shouldn't show up.

Comment (tomasbrod) [reply README.md:88]: It is probably because, you merged staging into sb-midpoint-rounding.

**Comment by iFoggz** [comment]
good work. :)

**Comment by gridcoin** [comment]
Great job.

Will be creating a leisure release ASAP tonight.

Rob

**Comment by iFoggz** [comment]
thanks @gridcoin for the update! cheers.

**Pull request merged**